docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<p>I would add that if we wanted we could derive the survival function, but we'll use scipy because it's already there. I'm saying this because I found it surprising when I saw <code>scipy.stats.sf</code> in the text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>